libevmasm/AssemblyItem.cpp

```diff
@@ -402,6 +405,18 @@ std::string AssemblyItem::computeSourceMapping(
 			}
 		}
 
+		// Append empty items if this AssignImmutable was referenced more than once.
```

Comment:
Can you extract this into a function of `AssemblyItem`?

Reply:
I mean a function that returns the number of opcodes generated from the assembly item.